SportsHub/SportsHub.DAL/Repository/UserRepository.cs

```diff
@@ -37,5 +37,21 @@ public async Task SaveUserAsync(User user)
                 .Include(x => x.Role)
                 .FirstOrDefaultAsync();
         }
+
+        public async Task<bool> DeleteArticle(Article article)
+        {
+            bool findArticle = false;
+            
+            await foreach (var contextUser in _context.Users)
+            {
+                if (contextUser.Articles.Any(x => x.Id == article.Id))
+                {
+                    contextUser.Articles.Remove(article);
```

Comment:
actually, you should return User, not Article here

Reply:
but here i am searching in User`s collection of Articles and when i found Article with matching Id i delete it and return bool in the end

Reply:
i changed it now i dont return anything im just searching if a user has this article and if he has i delete it otherwhise i am not deleting anything